variantlib/constants.py

```diff
@@ -25,7 +25,7 @@
 VARIANTS_JSON_SCHEMA_URL = "https://variants-schema.wheelnext.dev/"
 VARIANTS_JSON_VARIANT_DATA_KEY: Literal["variants"] = "variants"
 
-VALIDATION_VARIANT_HASH_REGEX = re.compile(rf"[0-9a-f]{{{VARIANT_HASH_LEN}}}")
+VALIDATION_VARIANT_LABEL_REGEX = re.compile(r"[0-9a-z_]{1,8}")
```

Comment:
```suggestion
VALIDATION_VARIANT_LABEL_REGEX = re.compile(rf"[0-9a-z_]{1,{{VARIANT_HASH_LEN}}}")
```

This would be slightly better

Reply:
Fair enough - closing